test/fork/migrated-claims.js

```diff
@@ -0,0 +1,498 @@
+
+      console.log(`Redeeming payout in coverAsset ${expectedCoverAsset} for Cover with V2 ID: ${coverIdV2}`);
+      if (expectedCoverAsset === 0) {
+        // ETH
+        const ethBalanceBefore = await ethers.provider.getBalance(memberAddress);
+
+        console.log(`Current member balance ${ethBalanceBefore.toString()}. Redeeming claim ${claimId}`);
+
+        // redeem payout
+        await this.individualClaims.redeemClaimPayout(claimId);
+
+        const ethBalanceAfter = await ethers.provider.getBalance(memberAddress);
+
+        console.log(`Check correct balance increase`);
+        expect(ethBalanceAfter).to.be.equal(ethBalanceBefore.add(claimAmount).add(deposit));
+
+        const { payoutRedeemed } = await this.individualClaims.claims(claimId);
+        expect(payoutRedeemed).to.be.equal(true);
+      } else {
+        // DAI
+        const daiBalanceBefore = await this.dai.balanceOf(memberAddress);
+
+        console.log(`Current member balance ${daiBalanceBefore.toString()}.  Redeeming claim ${claimId}`);
+
+        // redeem payout
+        await this.individualClaims.redeemClaimPayout(claimId);
+
+        console.log(`Check correct balance increase`);
+        const daiBalanceAfter = await this.dai.balanceOf(memberAddress);
+        expect(daiBalanceAfter).to.be.equal(daiBalanceBefore.add(claimAmount));
+
+        const { payoutRedeemed } = await this.individualClaims.claims(claimId);
+        expect(payoutRedeemed).to.be.equal(true);
+      }
+
+      console.log(`Payout reedemed succesfully.`);
+    }
+  }
+
+  it('Migrate specific sherlock covers', async function () {
+    const coverIDsV1 = [8305];
+
+    const SHERLLOCK_GRACE_PERIOD = 35; // 35 days
+    const SHERLOCK_ID_V1 = '0x0000000000000000000000000000000000000029';
+
+    const claimAmounts = {
+      8305: parseEther('1116000'),
+    };
+
+    await migrateClaimAndRedeem.apply(this, [
+      { coverIDsV1, expectedGracePeriod: SHERLLOCK_GRACE_PERIOD, v1ProductId: SHERLOCK_ID_V1, claimAmounts },
+    ]);
+  });
+
+  it('Migrates, claims and reedeems existing FTX covers to V2', async function () {
+    const coverIDsV1 = [7907, 7881, 7863, 7643, 7598, 7572, 7542, 7313, 7134];
+
+    const FTX_GRACE_PERIOD = 120; // 120 days
+    const FTX_ID_V1 = '0xC57d000000000000000000000000000000000011';
+
+    await migrateClaimAndRedeem.apply(this, [
+      { coverIDsV1, expectedGracePeriod: FTX_GRACE_PERIOD, v1ProductId: FTX_ID_V1 },
+    ]);
+  });
+
+  it('Migrates normal Euler and Bright Union Euler covers to V2', async function () {
+    const BRIGHT_UNION_DISTRIBUTOR = '0x425b3A68f1FD5dE26b4B9F4Be8049E36406B187A';
+
+    const brightUnionDistributor = await ethers.getContractAt('IBrightUnionDistributor', BRIGHT_UNION_DISTRIBUTOR);
+
+    const simpleCoverIds = [7898, 8121, 8166, 8167, 8201, 8210, 8223, 8234, 8285, 8317];
+
+    // 8220 is close to expiry - skip the other tests and only run this one to get it to be within the right time range
```

Comment:
would need to alter the tests to allows checking all covers concurrently or reset fork state before each 

one can also just skip the others and test this block

the issue is when i advance time to get claims voted some covers expire like 8220